gdx/src/com/badlogic/gdx/graphics/g3d/environment/AmbientCubemap.java

```diff
@@ -141,10 +140,10 @@ public AmbientCubemap add (final Color color, final Vector3 point, final Vector3
 
 	@Override
 	public String toString () {
-		String result = "";
+		StringBuilder result = new StringBuilder();
 		for (int i = 0; i < data.length; i += 3) {
-			result += Float.toString(data[i]) + ", " + Float.toString(data[i + 1]) + ", " + Float.toString(data[i + 2]) + "\n";
+			result.append(data[i]).append(", ").append(data[i + 1]).append(", ").append(data[i + 2]).append("\n");
 		}
-		return result;
+		return result.toString();

```

Comment:
Again, `toString` used for debug, so not a great need for change. Though I personally find this more readable, because the ` + ` between `", "` are just a lot of characters to visually parse. That is probably just me though.